packages/muelu/README.md

```diff
@@ -0,0 +1,15 @@
+# MueLu: A package for multigrid based preconditioning
+
+
+## Copyright and License
+See muelu/COPYRIGHT, muelu/LICENSE, https://trilinos.github.io/license.html and individual file headers for additional information.
+
+
+## Questions? 
+Contact lead developers:
+
+* MueLu team         (GitHub handle: @trilinos/muelu)
+* Jonathan Hu        (GitHub handle: [jhux2](https://github.com/jhux2) or jhu@sandia.gov)
+* Andrey Prokopenko  (GitHub handle: [aprokop](https://github.com/aprokop)
+* Ray Tuminaro       (GitHub handle: [rstumin](https://github.com/rstumin) or rstumin@sandia.gov)
+* Tobias Wiesner     (GitHub handle: [tawiesn](https://github.com/tawiesn)
```

Comment:
Andrey and Tobias are not workin on MueLu anymore.

@jhux2 Is this the place, where you'd prefer to just put the MueLu team GitHub handle?

Reply:
@mayrmt That's right.  I see that @ccober6 already removed individual names from the other headers, which is great.  (I had been meaning to do that but never got around to it.)

Reply:
@mayrmt and @jhux2 , yeah, I can just leave the muelu handle.  There was not another copyright assertion done, so just leaving the previous copyright in-place.  This is just cleaning up the copyright notice and license, along with including the SPDX-License-Identifier.

Reply:
FYI, by cleaning up the headers, this was able to remove +30k lines of comments!

Reply:
@ccober6 You are going to jump into the Trilinos contributor lead for sure!

Reply:
Yeah, I will have touched almost every file and eliminated a lot of comment lines.  The Great Comment Killer!  :)